edu-blog-vue/components/AuthorBio.vue

```diff
@@ -13,8 +13,7 @@
           >
         </h5>
         <p>
```

Comment:
You could for example use the class `author-bio__text` here. that helps you with the styling (see comment in css)